openhands/agenthub/codeact_agent/codeact_agent.py

```diff
@@ -509,7 +509,16 @@ def _get_messages(self, state: State) -> list[Message]:
             )
             # do not add this for function calling
             if latest_user_message:
-                reminder_text = f'\n\nENVIRONMENT REMINDER: You have {state.max_iterations - state.iteration} turns left to complete the task. When finished reply with <finish></finish>.'
+                reminder_text = (
+                    '\n\nENVIRONMENT REMINDER:\n'
+                    f'- You have {state.max_iterations - state.iteration} turns left to complete the task\n'
+                    '- When finished reply with <finish></finish>\n'
+                    '\n<IMPORTANT>\n'
+                    '- You MUST generate only one action per turn!\n'
+                    '- A patch is a set of changes to the source code of the codebase that you are given\n'
+                    '- You MUST generate a patch that attempts to fix the issue described in the <pr_description>\n'
```

Comment:
Can we make this only available for SWE-Bench evaluation? This seems to be less useful for other general tasks?

Other than that i think I'm overall good with this change!